### ISSUE TITLE
[SCREENREADER]: No h1 heading on page

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-cms#9721]
[508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)
## Feedback framework
- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

This would be a **⚠️ Should**.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## User Story or Problem Statement
As a screen reader user, I expect to encounter a h1 on the page, so that I am clearly able to understand what the page is about.

## Details
Currently, the home page has no h1 at all. This can lead to confusion for screen reader users who might expect a h1 on the page to help determine what the page's main topic area is about.

## Acceptance Criteria
- [ ] Review context in Slack thread re: this ticket to understand discussion around semantic markup vs. visual styling: 
https://dsva.slack.com/archives/C52CL1PKQ/p1657138423423829
- [ ] Page has at least one h1

## Steps to Recreate
1. Open up developer tools in any desktop browser.
2. Search the HTML source code for a h1 element.

## Proposed Solution (if known)
A few possible solutions here:
1. Make the h2 in the first hero area a h1. This heading: "Access and manage your VA benefits and health care" This probably isn't ideal because that copy isn't really the full topic of the page.
2. Add a screen reader only heading, which includes the class `vads-u-visibility--screen-reader`.

## WCAG or Vendor Guidance (optional)
* [Accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)

**Comment by jcklpe** [comment]
Let me see if I understand everything right:

*   First picture is how it is currently
    *   It's an h1 but styled like an h2
*   Second picture is the redesign
    *   It's an h2 but semantically should be an h1

Solution to me seems to be: make it an h1, style it like an h1. Form follows function and all that. Can't imagine that a h1styling would really be that big a deal, and semantics take precedence IMO. When you start changing the design to be different from the semantics it indicates to me a weakness in the design to begin with.

**Comment by mmiddaugh** [comment]
@TiffanyPender, will you please validate this change resolves the defect, when you are able? Thank you!

**Comment by TiffanyPender** [comment]
Verified - page now includes an H1 so that screen readers can understand the context of the page. Looks good! @mmiddaugh 

<img width="1422" alt="VA-home page - H1 included on page" src="https://user-images.githubusercontent.com/25351510/181832056-22bc6b0f-de58-4303-939c-0f8b0340dfcc.png">
